components/AnalysisForm/index.js

```diff
@@ -191,6 +202,51 @@ export default function AnalysisForm({ typeOfAnalysis }) {
             </>
           );
         }
+        if (analysisKey === "comparison") {
+          return (
+            <EntryContent>
+              <ContentHeadline htmlFor={analysisKey}>
+                {analysisHeadlines[analysisKey].title}
+              </ContentHeadline>
+              <p>{analysisHeadlines[analysisKey].description}</p>
+              <OutcomeReachedSection>
+                <input
+                  type="radio"
+                  id="comparisonYes"
+                  name="comparison"
+                  value={true}
+                  checked={outcomeReached === true}
+                  onChange={onOptionChange}
+                />
+                <label htmlFor="comparisonYes">Ja</label>
+              </OutcomeReachedSection>
+              <OutcomeReachedSection>
+                {" "}
```

Comment:
What do you need the space for?

Reply:
oh, i think that was a mistake made by prettier :) 
thanks for seeing it!